src/backend/server.ts

```diff
@@ -0,0 +1,89 @@
+import Fastify, { type FastifyRequest } from 'fastify';
+import { readFile } from 'node:fs/promises';
+import { config } from './lib/config.js';
+import { rootDir } from './lib/constants.js';
+import { DocumentHandler } from './lib/documentHandler.js';
+import type { FastifyRequestGeneric } from './lib/types.js';
+
+// Use dynamic imports to ensure only one of the stores gets loaded
+const preferredStore =
+	config.storage.type === 'file'
+		? new (await import('./lib/document_stores/file.js')).FileDocumentStore(config.storage)
+		: new (await import('./lib/document_stores/redis.js')).RedisDocumentStore(config.storage);
+
+// Load all static documents
+for (const [name, path] of Object.entries(config.documents)) {
+	const data = await readFile(path, 'utf-8');
+
+	if (data) {
```

Comment:
This is always truthy.

Reply:
Yes... so this has to be kept...?

Reply:
It's... not clear enough, maybe write a comment? Or if you use `@sapphire/utilities`, use `isNullishOrEmpty`.